pkg/controller/backup_finalizer_controller.go

```diff
@@ -209,3 +257,64 @@ func (r *backupFinalizerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&velerov1api.Backup{}).
 		Complete(r)
 }
+
+// waitVolumeSnapshotReadyToUse is used to wait VolumeSnapshot turned to ReadyToUse.
+// Waiting for VolumeSnapshot ReadyToUse to true is time consuming. Try to make the process parallel by
+// using goroutine here instead of waiting in CSI plugin, because it's not easy to make BackupItemAction
+// parallel by now. After BackupItemAction parallel is implemented, this logic should be moved to CSI plugin
+// as https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/100
+func (r *backupFinalizerReconciler) waitVolumeSnapshotReadyToUse(ctx context.Context,
```

Comment:
You don't need these funcs anymore since you're calling the version in backup_controller.go